atecc608a_se.h

```diff
@@ -23,8 +23,33 @@
 #ifndef ATECC608A_SE_H
 #define ATECC608A_SE_H
 
-#include "psa/crypto_se_driver.h"
+#include "psa/crypto.h"
```

Comment:
You shouldn't need to include `crypto.h` here. Why isn't `crypto_se_driver.h` sufficient? If it's missing something, that's probably a bug in that header.

Reply:
amended.